CrowdStrike/crowdstrike_telemetry/schemas.py

```diff
@@ -0,0 +1,36 @@
+"""Contains schemas for sqs messages."""
+from pydantic import BaseModel
+
+
+class FileInfoSchema(BaseModel):
+    """File info schema that we receive from sqs."""
+
+    path: str
+
+
+class CrowdStrikeEventSchema(BaseModel):
```

Comment:
I suggest calling this class `CrowdstrikeNotificationSchema`; because messages are notifications.